packages/diagram_capture/lib/diagram_capture.dart

```diff
@@ -441,6 +442,8 @@ class DiagramController {
         }
       }
 
+      if(gestureCallback != null)
```

Comment:
trivial nit: put a space between keywords/identifiers and `(` unless it's a method call